src/main/java/io/github/externschool/planner/entity/profile/Gender.java

```diff
@@ -0,0 +1,17 @@
+package io.github.externschool.planner.entity.profile;
```

Comment:
i'd use a new package, something like "io.github.externschool.planner.enum"

Reply:
Enum is closely related to Student
 and we can not name package as "enum": naming convention error)